src/main/java/core/storage/Storage.java

```diff
@@ -0,0 +1,15 @@
+package core.storage;
+
+import core.model.Manufacturer;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<Long, Manufacturer> storage = new HashMap<>();
```

Comment:
```suggestion
    public static final Map<Long, Manufacturer> manufacturers = new HashMap<>();
```
there will be cars and drivers as well

Reply:
Ok